source/catch/main.cc

```diff
@@ -28,16 +28,30 @@
 #include "../test/pgg_mode_test/PGGDataNodes.test.cc"
 #include "../test/pgg_mode_test/PGGWorld.test.cc"
 
-#include "../test/sgp_mode_test/GenomeLibrary.test.cc"
-#include "../test/sgp_mode_test/SGPWorld.test.cc"
-#include "../test/sgp_mode_test/SGPHost.test.cc"
-#include "../test/sgp_mode_test/SGPSymbiont.test.cc"
-#include "../test/sgp_mode_test/StressHost.test.cc"
-#include "../test/sgp_mode_test/HealthHost.test.cc"
-#include "../test/sgp_mode_test/SGPDataNodes.test.cc"
-#include "../test/sgp_mode_test/Tasks.test.cc"
-#include "../test/sgp_mode_test/Instructions.test.cc"
-#include "../test/sgp_mode_test/SGPWorldSetup.test.cc"
+
+#include "../test/sgp_mode_test/unit_tests/unit_GenomeLibrary.test.cc"
+#include "../test/sgp_mode_test/unit_tests/unit_HealthHost.test.cc"
+#include "../test/sgp_mode_test/unit_tests/unit_SGPSymbiont.test.cc"
+#include "../test/sgp_mode_test/unit_tests/unit_SGPWorld.test.cc"
+#include "../test/sgp_mode_test/unit_tests/unit_SGPWorldSetup.test.cc"
+#include "../test/sgp_mode_test/unit_tests/unit_Tasks.test.cc"
+
+
+#include "../test/sgp_mode_test/functional_tests/GenomeLibrary_SGPWorld.test.cc"
+#include "../test/sgp_mode_test/functional_tests/HealthHost_Instructions.test.cc"
+#include "../test/sgp_mode_test/functional_tests/SGPDataNodes.test.cc"
+#include "../test/sgp_mode_test/functional_tests/SGPHost_Tasks.test.cc"
+#include "../test/sgp_mode_test/functional_tests/SGPHost.test.cc"
+#include "../test/sgp_mode_test/functional_tests/SGPSymbiont_Tasks.test.cc"
+#include "../test/sgp_mode_test/functional_tests/SGPSymbiont.test.cc"
+#include "../test/sgp_mode_test/functional_tests/SGPWorld_StressHost.test.cc"
+#include "../test/sgp_mode_test/functional_tests/SGPWorld.test.cc"
```

Comment:
Similar to the other comments, if this is a specific complicated thing, it would be good to be more specific in the name of the file